src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -45,7 +45,7 @@
           />
 
         <!-- SP add buttons (conversion filing only) -->
-        <div v-if="isTypeSoleProp && isConversionFiling" class="mt-8">
+        <div v-if="isTypeSoleProp && isConversionFiling && !hasMinimumProprietor" class="mt-8">
```

Comment:
Only show buttons if we don't have minimum proprietor.

Ditto below for partners.